### PR TITLE
implement design for welcome screen.

### PR DESCRIPTION
**Pull request by StefanKosev1**

### CODE REVIEW

File: app/src/main/res/layout/activity_welcome.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    android:padding="20dp">
+
+
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="40dp"
+        android:layout_marginStart="10dp"
+        android:layout_marginTop="10dp"
+        android:layout_marginEnd="10dp"
+        android:text="This product uses the TMDb API but is not endorsed or certified by TMDB"
```
Comment (tkosev123) [inline app/src/main/res/layout/activity_welcome.xml]: This text should be extract in string resources file, applies to all text in this file

File: app/src/main/res/layout/activity_welcome.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+    <Button
+        android:id="@+id/myButton"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_marginTop="8dp"
+        android:layout_weight="1"
+        android:background="@drawable/my_button"
+        android:text="Movies"
+        android:textColor="@color/white"
```
Comment (tkosev123) [inline app/src/main/res/layout/activity_welcome.xml:27]: We should define our own colour and use those not the build in.

File: app/src/main/res/layout/activity_welcome.xml (hunk 1 above)
Comment (StefanKosev1) [reply app/src/main/res/layout/activity_welcome.xml]: Done :Д

File: app/src/main/res/layout/activity_welcome.xml (hunk 2 above)
Comment (StefanKosev1) [reply app/src/main/res/layout/activity_welcome.xml:27]: Done

File: app/src/main/res/layout/activity_welcome.xml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        android:textSize="20sp" />
+
+    <Button
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_marginTop="20dp"
+        android:layout_weight="1"
+        android:background="@drawable/my_button"
+        android:text="@string/welcome_welcomeTvShowsButton"
```
Comment (tkosev123) [inline app/src/main/res/layout/activity_welcome.xml]: the name of the string should use underscore notation please change it to welcome_tv_show_button_text and update all other key for string resources

Comment (StefanKosev1) [reply app/src/main/res/layout/activity_welcome.xml]: done

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<resources>
+    <string name="app_name">TMDB</string>
+    <string name="welcome_title">This product uses the TMDb API but is not endorsed or certified by TMDB</string>
+    <string name="welcome_welcome_Movies_Button">Movies</string>
+    <string name="welcome_welcome_TvShows_Button">TV Shows</string>
+    <string name="welcome_welcome_Favorites_Button">Favorites</string>
```
Comment (tkosev123) [inline app/src/main/res/values/strings.xml]: We should not use upper cases in our keys

**Pull request merged**